resources/seccomp/aarch64-unknown-linux-musl.json

```diff
@@ -1068,22 +417,9 @@
                     }
                 ]
             },
-            {
-                "syscall": "mmap",
-                "comment": "Used by the VirtIO balloon device",
```

Comment:
nit: These are rather used for allocating memory for guest memory regions not the balloon device in particular, right?

Reply:
In this case, after the seccomp filters are installed, this `mmap` is only used by the balloon device, in this specific case: https://github.com/firecracker-microvm/firecracker/blob/67cbafbc51e41a3f63b4481a0409206b623c08c0/src/devices/src/virtio/balloon/utils.rs#L87